src/main/java/net/sf/jabref/logic/exporter/BibTeXMLExportFormat.java

```diff
@@ -141,7 +140,7 @@ private void createMarshallerAndWriteToFile(File file, String resultFile) throws
     }
 
     /**
-     * Contains same logic as the parse method, but inbook needs a special treatment, because
+     * Contains same logic as the {@link parse()} method, but inbook needs a special treatment, because
```

Comment:
To link to a method you need to use the following annotation `{@link #parse()}`, But the rest is good to me.